docs/javascript/browser-apis-and-jquery.md

```diff
@@ -1143,3 +1143,175 @@ When you select an element from the DOM, there are many properties that live on
 ![attr example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/attr.gif?raw=true){ width="100%" }
 <figcaption></figcaption>
 </figure>
+
+
+## Changing an element's content
+
+#### `textContent` property sets or return the text content of an element
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Home</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+
+  <div class="object" id="one">One</div>
+  <div class="object" id="two">Two</div>
+
+</div>
+<script>
+   function changeText() {
+        var element = document.querySelector('#one');
+        // Here we are setting the text content of id `one` to the new text
+        element.textContent = "New Text";
+    }
+         
+    changeText()
+
+</script>
+</html>
+
+```
+<figure markdown>
+![textContent example](https://github.com/codewizardshq/docs/blob/main/docs/assets/browser-apis-and-jquery/textContent.png?raw=true){ width="100%" }
+<figcaption></figcaption>
+</figure>
+
+#### `innerHtml` property sets or returns the HTML of an element
+
+```html
+<!DOCTYPE html>
+<html>
+
+<head>
+<title>Home</title>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+</head>
+
+<body>
+
+<div>
+
+ <div class="container">
+    <div class="object" id="one">One</div>
+    <div class="object" id="two">Two</div>
+  </div>   
+
+</div>
+<script>
+     // Here we are accessing the HTML content of an element 
+      var element = document.querySelector('#one').innerHTML;
```

Comment:
I honestly have no idea why CWHQ uses `innerHTML` for things like this, but let's not follow that pattern in the docs. I've got some standards work happening on the Curriculum team that will hopefully fix these issues in courses.

We should only use `innerHTML` when we want to edit the HTML of an element, not the text content. If we want to edit text content, we should use `textContent`.